<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_auto</class>
 <widget class="QWidget" name="add_auto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Марка и модель</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_marka">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_god">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Госномер</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gosnomer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_VIN">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Состояние</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ispraven">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>109</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Исправен</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_remont">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>109</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>На ремонте</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
